sui/token_bridge/sources/datatypes/string32.move

```diff
@@ -96,6 +97,18 @@ module token_bridge::string32 {
         string::utf8(bytes)
     }
 
+    public fun to_ascii_string(s: &String32): ascii::String {
```

Comment:
we should document what this function does (and what it doesn't do). currently it fails on strings that contain non-ascii characters, which the symbol might well do, since the token bridge spec allows utf8 strings here. Currently it means that tokens with unicode characters in their symbol name are not supported. we can either document this, or fix somehow (e.g. by removing non-ascii characters)